product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/index.mdx

```diff
@@ -163,7 +163,15 @@ Exclude these Oracle systems schemas while generating the SQL dump file.
 -   Directories
 -   Users
 -   RLS policy
--   Queues
+-   Queues *
+-   Library
+-   Alter Function
```

Comment:
I don't think we should list Alter Function, Alter Package, and Alter Type here like this. They are not object types. Rather, we should probably have a note that indicates that with the exception of ALTER TABLE statements that add constraints, Migration Portal does not process ALTER statements that may added to the DDL files generated by the Oracle data pump utilities.
@umangjtank can you confirm that no other ALTER statements other than ALTER TABLE statements that create constraints are processed by Migration Portal?

Reply:
Added the following in the known issues, limitations, and notes page.

under Migration Portal:
"Except for ALTER TABLE and ALTER TRIGGER, Migration Portal does not process any other ALTER statement in the DDL."

under Oracle Data Pump utilities:
"The DDL generated by Oracle Data Pump utilities might contain ALTER STATEMENTS such as ALTER FUNCTION, ALTER PACKAGE, and ALTER TYPE. Migration Portal does not process these statements."